jib-core/src/test/java/com/google/cloud/tools/jib/configuration/BuildConfigurationTest.java

```diff
@@ -172,14 +171,14 @@ public void testBuilder_default() {
             .setTargetImageConfiguration(targetImageConfiguration)
             .build();
 
-    Assert.assertEquals(V22ManifestTemplate.class, buildConfiguration.getTargetFormat());
+    Assert.assertEquals(
+        BuildConfiguration.DEFAULT_TARGET_FORMAT, buildConfiguration.getTargetFormat());
```

Comment:
I think we can leave these using the actual value and make the constants private.